src/js/tabs/pid_tuning.js

```diff
@@ -182,6 +182,9 @@ TABS.pid_tuning.initialize = function (callback) {
                 var checked = $(this).is(':checked');
                 if (checked) {
                     $('.antigravity input[name="itermAcceleratorGain"]').val(Math.max(ADVANCED_TUNING.itermAcceleratorGain / 1000, 1.1));
+                    if (semver.gte(CONFIG.apiVersion, "1.43.0")) {
+                        $('.antigravity input[name="itermAcceleratorGain"]').val(Math.max(ADVANCED_TUNING.itermAcceleratorGain / 1000, 3.5));
+                    }
```

Comment:
I think this will be a bit easier to read:

```
let itermAcceleratorGain = ADVANCED_TUNING.itermAcceleratorGain / 1000;
if (semver.gte(CONFIG.apiVersion, "1.43.0")) {
    itermAcceleratorGain = Math.max(itermAcceleratorGain, 3.5);
} else {
    itermAcceleratorGain = Math.max(itermAcceleratorGain, 1.1);
}
input[name="itermAcceleratorGain"]').val(itermAcceleratorGain);
```

Reply:
Another option ;)
```
const MAX_ACCELERATOR_GAIN = semver.gte(CONFIG.apiVersion, "1.43.0") ? 3.5 : 1.1;
const itermAcceleratorGain = Math.max(ADVANCED_TUNING.itermAcceleratorGain / 1000, MAX_ACCELERATOR_GAIN);
input[name="itermAcceleratorGain"]').val(itermAcceleratorGain);
```

Reply:
Interesting @McGiverGim , How we can add more rules in the future with this method? with another const?

Reply:
If in the future are more rules, we will need to change it. 
Is not a problem, we try to make the code with good maintainability, but we don't sacrifice current code thinking in what can happen in the future. The most probably is that we go wrong with it because we don't know in what sens this will change.